### ISSUE TITLE
ArgumentError - wrong number of arguments (given 3, expected 7..9)

### ISSUE THREAD

**Issue by guxs1129** [CocoaPods/cocoapods-packager#222]
Have problem with packager 1.5.0 & cocoapods 1.6.0beta when package a framework.And use cocoapods 1.5.3,it works fine.
### Command

```
/usr/local/bin/pod package ../project/eagle-base/EagleSDK.podspec --dynamic --force --spec-sources=ssh://git@192.168.2.94:10022/repertory/cocoapods/Specs.git,https://github.com/CocoaPods/Specs.git
```

### Report

* What did you do?
* What did you expect to happen?
* What happened instead?

### Stack
```
   CocoaPods : 1.6.0.beta.1
        Ruby : ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-darwin17]
    RubyGems : 2.6.14
        Host : Mac OS X 10.13.6 (17G65)
       Xcode : 9.4.1 (9F2000)
         Git : git version 2.15.2 (Apple Git-101.1)
Ruby lib dir : /usr/local/Cellar/ruby/2.4.2_1/lib
Repositories : 2-repertory-cocoapods-specs - http://192.168.2.94:10080/repertory/cocoapods/Specs @ bdc3db7c2cfa80b42afa13bd3d3d75a2ce348ec5
               :path=> -  @ 1854b6f725ec7d2e12c422b80b97496216475164
               :path=>' -  @ 1854b6f725ec7d2e12c422b80b97496216475164
               Linkstec - ssh://git@192.168.2.94:10022/repertory/cocoapods/Specs.git @ 9c4d16b599de849a043e29fc8fba20e223fd333d
               master - https://github.com/CocoaPods/Specs.git @ 24f220eccbcd9d24ef0d83f73fa626670cd25164
```

### Plugins

```
cocoapods-deintegrate : 1.0.2
cocoapods-packager    : 1.5.0
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.3.1
cocoapods-try         : 1.1.0
```

### Error

```
ArgumentError - wrong number of arguments (given 3, expected 7..9)
/usr/local/lib/ruby/gems/2.4.0/gems/cocoapods-1.6.0.beta.1/lib/cocoapods/target/pod_target.rb:62:in `initialize'
/usr/local/lib/ruby/gems/2.4.0/gems/cocoapods-packager-1.5.0/lib/cocoapods-packager/pod_utils.rb:161:in `new'
/usr/local/lib/ruby/gems/2.4.0/gems/cocoapods-packager-1.5.0/lib/cocoapods-packager/pod_utils.rb:161:in `build_dynamic_target'
/usr/local/lib/ruby/gems/2.4.0/gems/cocoapods-packager-1.5.0/lib/cocoapods-packager/pod_utils.rb:131:in `install_dynamic_pod'
/usr/local/lib/ruby/gems/2.4.0/gems/cocoapods-packager-1.5.0/lib/pod/command/package.rb:83:in `build_in_sandbox'
/usr/local/lib/ruby/gems/2.4.0/gems/cocoapods-packager-1.5.0/lib/pod/command/package.rb:100:in `block in build_package'
/usr/local/lib/ruby/gems/2.4.0/gems/cocoapods-packager-1.5.0/lib/pod/command/package.rb:99:in `each'
/usr/local/lib/ruby/gems/2.4.0/gems/cocoapods-packager-1.5.0/lib/pod/command/package.rb:99:in `build_package'
/usr/local/lib/ruby/gems/2.4.0/gems/cocoapods-packager-1.5.0/lib/pod/command/package.rb:66:in `run'
/usr/local/lib/ruby/gems/2.4.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
/usr/local/lib/ruby/gems/2.4.0/gems/cocoapods-1.6.0.beta.1/lib/cocoapods/command.rb:52:in `run'
/usr/local/lib/ruby/gems/2.4.0/gems/cocoapods-1.6.0.beta.1/bin/pod:55:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'
```

**Comment by Zss1990** [comment]
me too

**Comment by amorde** [comment]
Fixed in #240
